### DISCUSSION TITLE
¿Bug? When there is only one result, it does not scrappe.

### DISCUSSION THREAD

**Discussion by MCancemi** [omkarcloud/google-maps-scraper#92] [General]
When I perform a search that has only one direct result, which when I search for it on Google goes to the company file, it does not return results in the CSV or JSON.

**Comment by Chetan11-dev** [comment]
Code to reproduce?

**Reply by MCancemi** [reply]
from src import Gmaps

love_it_star_it = '''Love It? Star It! ⭐ https://github.com/omkarcloud/google-maps-scraper/'''

queries = [
   "hisn yakka en Yecla, murcia"
]

Gmaps.places(queries, max=5)

**Comment by Chetan11-dev** [comment]
Fixed, Please follow these Steps. 

⿡ Clone the Magic 🧙‍♀:
```
git clone https://github.com/omkarcloud/google-maps-scraper
cd google-maps-scraper
```

⿢ Install Dependencies 📦:
```
python -m pip install -r requirements.txt
```

⿣ Let the Rain of Google Map Leads Begin 😎:
```
python main.py 
```

And Run the following command to update packages:

```
python -m pip install botasaurus --upgrade
```

**Comment by Chetan11-dev** [comment]
Also, would appreciate if you could star the project :)
